README.md

```diff
@@ -22,25 +22,41 @@ wip.
 
 wip.
 
-## Build
+## Build the Project
 
-Use the maven wrapper.
-
-To build all modules.
+Use the maven or maven wrapper to build.
 ```
 ./mvnw clean package
 ```
 
+The project requires java version 8.
+
 Artifacts will appear in each sub modules' target directory.
 
-## Deploy the Ranger Service
+## Register Ranger GCS Service to Apache Ranger Server
 
-wip.
+There must be an up-and-running Apache Ranger Server running on localhost.
+The scripts bellow will register the Ranger GCS service to the Ranger server.
+
+More information about [Dataproc Ranger Component](https://cloud.google.com/dataproc/docs/concepts/components/ranger).
```

Comment:
Let's move this sentence directly below the "There must be a Apache Ranger Server running..." sentence.
Also,  change it to something like "If you are using Google Cloud Dataproc, you can install the [Ranger optional componet](...)".